test/decl/var/properties.swift

```diff
@@ -472,6 +472,18 @@ extension ProtocolWithExtension1 {
   static var fooExtStatic = 4 // expected-error{{static stored properties not supported in protocol extensions}}
 }
 
+protocol ProtocolWithExtension2 {
+  var bar: String { get }
+}
+
+struct StructureImplementingProtocolWithExtension2: ProtocolWithExtension2 {
+  let bar: String
+}
+
+extension ProtocolWithExtension2 {
+  static let baz: Foo = StructureImplementingProtocolWithExtension2(bar: "baz") // expected-error{{static stored properties not supported in protocol extensios}}
```

Comment:
`Foo` is not a well-defined type

Reply:
Sorry, forgot to rename. Fixed on 69cafb5.